### PR TITLE
chore(tests): added coverage for image deletion and renamed image e2e test file

### PR DESCRIPTION
**Pull request by xbabalov**
### What does this PR do?
Implements test cases for deleting image. Renames the pull-image-smoke.spec.ts to image-smoke.spec.ts.

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop/issues/3552

### How to test this PR?
yarn test:e2e:smoke

### CODE REVIEW

File: tests/src/model/pages/images-page.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class ImagesPage extends PodmanDesktopPage {
     }
   }
 
-  protected async imageExists(name: string): Promise<boolean> {
+  async imageExists(name: string): Promise<boolean> {
```
Comment (benoitf) [inline tests/src/model/pages/images-page.ts]: please do not call this method, it was flagged as protected as it does not wait
`waitForImageExists` has to be used instead

File: tests/src/image-smoke.spec.ts [hunk 2]
```diff
@@ -65,3 +68,20 @@ describe('Image pull verification', async () => {
...
+
+    await playExpect(imageDetailPage.deleteButton).toBeVisible();
+    await imageDetailPage.deleteButton.click();
+  });
+
+  test('Verify image was deleted', async () => {
+    const imagesPage = new ImagesPage(page);
+    await playExpect(imagesPage.heading).toBeVisible();
+
+    const imageExists = await imagesPage.imageExists('quay.io/podman/hello');
```
Comment (benoitf) [inline tests/src/image-smoke.spec.ts]:
```suggestion
    const imageExists = await imagesPage.waitForImageExists('quay.io/podman/hello');
```

**Review by benoitf — CHANGES_REQUESTED** [review]
need to use the waiting method for imageExists

File: tests/src/image-smoke.spec.ts [hunk 3]
```diff
@@ -65,3 +68,20 @@ describe('Image pull verification', async () => {
     await playExpect(imageDetailPage.inspectTab).toBeVisible();
   });
 });
+
+describe('Image delete verification', async () => {
```
Comment (odockal) [inline tests/src/image-smoke.spec.ts]: We could probably change the first describe to a 'Image workflow verification', and then remove this, then unite two test methods into one?

**Review by odockal — CHANGES_REQUESTED** [review]

**Comment by xbabalov** [comment]
Thank you for the review @benoitf @odockal, everything requested should be fixed now.

**Review by odockal — APPROVED** [review]

**Comment by odockal** [comment]
@xbabalov Please rebase, and if there are not objections I would merge the PR. All review comments were incorporate. @benoitf

**Review by benoitf — APPROVED** [review]
I confirm changes were handled

thanks

**Pull request merged**